FE/baseball/src/Components/Game/SquadBoard/SquadTable/SquadTable.jsx

```diff
@@ -0,0 +1,34 @@
+import SquadTableHeader from "./SquadTableHeader";
+import SquadTableBody from "./SquadTableBody";
+import { SquadBoard as S } from "@/Components/Game/GameStyles";
+
+const SquadTable = ({ teamName, selectedTeam, isDefenseTeam, squads }) => {
+  let currentPlayerNumber = 3;
+
+  const newSquads = squads.map((v, index) => {
+    if (isDefenseTeam === teamName) {
+      // 수비팀일 경우 투수를 하이라이트
+      if (v.position === "pitcher") {
+        return { ...v, isCurrentPlayer: true };
+      } else {
+        return { ...v, isCurrentPlayer: false };
+      }
```

Comment:
크게 다를 것 없는데 if문을 안쓰고도 처리할 수 있을 것 같네요 ㅎㅎ
```suggestion
        return { ...v, isCurrentPlayer: v.position === "pitcher" };
```